cmake/cblas.cmake

```diff
@@ -83,20 +83,6 @@ else()
   set(REFERENCE_CBLAS_LIB_SEARCH_PATHS ${REFERENCE_CBLAS_ROOT}/lib)
 endif()
 
-find_path(REFERENCE_CBLAS_INCLUDE_DIR NAMES cblas.h PATHS
-        ${REFERENCE_CBLAS_INCLUDE_SEARCH_PATHS})
-find_library(REFERENCE_CBLAS_LIBRARY NAMES cblas PATHS
-        ${REFERENCE_CBLAS_LIB_SEARCH_PATHS})
-
-if(REFERENCE_CBLAS_INCLUDE_DIR AND REFERENCE_CBLAS_LIBRARY)
```

Comment:
why here delete these lines?

Reply:
Added comments:
```
# TODO(typhoonzero): I deleted those REFERENCE_CBLAS_INCLUDE_DIR things
# that may cause builds fail if we installed other cblas libs like "veclib",
# in paddle/fluid/operators/math/blas.h which will throw "Not Implemented"
# for other cblas. Openblas and MKL are supported only.
```

Reply:
Now indeed only Openblas and MKL are supported. We don't test other cblas libs. Why should you install and use other cblas libs?

Reply:
The other software installed veclib as a dependency. So I deleted this

Reply:
Though other software installed veclib, they can use MKL/openblas as well.

Reply:
If we need to keep this for future adding libraries, can add a switch here to determine whether to do `find_library`.